### DISCUSSION TITLE
Access token

### DISCUSSION THREAD

**Discussion by jet10000** [flet-dev/flet#396] [General]
Where should I store the access token for the user session for web?

**Comment by jet10000** [comment]
```
page.client_storage.set("access_token", "<token>")
```

https://github.com/flet-dev/flet/pull/288

**Reply by FeodorFitsner** [reply]
It's a right place, but don't put it there unencrypted as it could be easily read by the user!

**Reply by FeodorFitsner** [reply]
...and do not post any tokens on public forums please :)

**Reply by jet10000** [reply]
i see, thanks
